.github/workflows/perf_micro.yml

```diff
@@ -48,7 +48,17 @@ jobs:
           ( github.event_name == 'pull_request' &&
```

Comment:
I suppose we can't just drop this library since we build tests unconditionally for non-Debug builds. So, to avoid build failures, we must install this library for our builds.

Reply:
Okay.